### ISSUE TITLE
Reproduction issues

### ISSUE THREAD

**Issue by wang90063** [ds4dm/branch-search-trees#9]
Hello! We have  been reproducing the results of your insightful paper entilted " Parameterizing Branch-and-Bound Search Trees to 
Learn Branching Policies " and encountered some issues. First of all, we appreciate that you publish your code and data in the repo. 
And we utilize the data from the repo and the hyperparameters from the model file trying to reproduce retrain the model. The val/test
 acc of  NT and val acc of TG are similar to the results reported in the paper. But, the test acc of TG is 74.17%.
We are looking forward to discussing with you.
    ps: 

**our model hyperparam:**
-         Namespace(depth=5, dim_reduce_factor=2, dropout=0.0, eval_batchsize=500, hidden_size=64, infimum=8, lr=0.01, lr_decay_factor=0.1, lr_decay_schedule=[20, 30], momentum=0.9, norm='none', num_epochs=40, opt='adam', out_dir='trained-models/TreeGatePolicy_hidden64_depth5_lr0.01', policy_type='TreeGatePolicy', seed=0, test_h5_path='data/test.h5', top_k=[2, 3, 5, 10], train_batchsize=32, train_h5_path='data/train.h5', use_gpu=True, val_h5_path='data/val.h5', weight_decay=1e-05)

**model hyperparam from repo:**
-         Namespace(depth=5, dim_reduce_factor=2, dropout=0.0, eval_batchsize=500, hidden_size=64, infimum=8, lr=0.01, lr_decay_factor=0.1, lr_decay_schedule=[20, 30], momentum=0.9, norm='none', num_epochs=40, opt='adam', out_dir='trained-models/TreeGatePolicy_hidden64_depth5_lr0.01', policy_type='TreeGatePolicy', seed=0, test_h5_path='data/test.h5', top_k=[2, 3, 5, 10], train_batchsize=32, train_h5_path='data/train.h5', use_gpu=True, val_h5_path='data/val.h5', weight_decay=1e-05)

      
Best,
Zhi
